synthtool/languages/java.py

```diff
@@ -199,7 +199,8 @@ def _common_generation(
     s.copy(
         [library / f"grpc-google-{cloud_prefix}{service}-{version}{suffix}/src"],
         f"grpc-google-{cloud_prefix}{destination_name}-{version}/src",
-        required=True,
+        # For REST-only clients, like java-compute, gRPC artifact does not exist
+        # required=True,
```

Comment:
Can we preserve the default behavior and override with optional arguments? This is a nice confidence check to ensure that we didn't typo something (identifying a no-op vs. not copying files).

Reply:
Made it a function of diregapic parameter.